src/modules/languages/php.nix

```diff
@@ -162,6 +162,22 @@ in
       '';
     };
 
+    packages = lib.mkOption {
+      type = lib.types.submodule ({
+        options = {
+          composer = lib.mkOption {
+            type = lib.types.nullOr lib.types.str;
+            default = "composer";
```

Comment:
the type should be `types.package` and the default `cfg.package.packages.composer`.

Reply:
I have no idea how it should work with package 🤔 . I need to point to the `cfg.package.packages.XXXXX`

Reply:
This should work:

```nix
config = {
...
  languages.php.composer = lib.mkDefault cfg.pacakge.packages.composer;
...
}
```